R/utils.R

```diff
@@ -185,12 +185,15 @@ SparseMMFromFactor <- function(thefactor) {
 }
 
 
-## slm.fit.csr with a fix
+## slm.fit.csr with a fix and using lm.fit backend
 ##
 ## SparseM's slm.fit.csr has a bug for intercept only models
 ## (admittedly, these are generally a little silly to be done as a
 ## sparse matrix), but in order to avoid duplicate code, if
 ## everything is in a single strata, we use the intercept only model.
+##
+## In this maintenance release, this function takes SparseM csr matrix x and
```

Comment:
Despite appreciating the spirit of this inline comment, I'm going to suggest taking it out.  (In case updating the roxygen block gets forgotten when the code gets updated.)

Reply:
Also made this change.